### PR TITLE
Use VS code debugging icons in debugger

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Screen Shot 2021-02-10 at 10 04 06 AM
Screen Shot 2021-02-10 at 10 04 15 AM

### CODE REVIEW

**Comment by jacob314** [comment]
Third party content needs to be under the third_party directory in its own packages.

File: packages/devtools_app/pubspec.yaml [hunk 1]
```diff
@@ -24,7 +24,9 @@ dependencies:
   # Pin ansicolor to version before pre-NNBD version 1.1.0, should be ^1.0.5
   # See https://github.com/flutter/devtools/issues/2530 
   ansicolor: 1.0.5
-  async: ^2.0.0 
+  async: ^2.0.0
+  codicon:
+    path: ../../third_party/packages/codicon
```
Comment (kenzieschmoll) [inline packages/devtools_app/pubspec.yaml:29]: will update this to 0.0.1 once `codicon` is published

**Comment by kenzieschmoll** [comment]
Should we consider publishing this package outside of the devtools repo?

**Review by jacob314 — APPROVED** [review]
lgtm

**Pull request merged**